frame/balances/Cargo.toml

```
@@ -8,7 +8,7 @@ edition = "2018"
 serde = { version = "1.0.101", optional = true }
 safe-mix = { version = "1.0.0", default-features = false }
 codec = { package = "parity-scale-codec", version = "1.0.0", default-features = false, features = ["derive"] }
-rstd = { package = "sp-std", path = "../../primitives/sr-std", default-features = false }
+sp-std = { package = "sp-std", path = "../../primitives/std", default-features = false }
```

```suggestion
sp-std = { path = "../../primitives/std", default-features = false }
```

would recommend we remove the `package = ...` everywhere as well.